Documentation/concepts.rst

```diff
@@ -610,6 +610,15 @@ This is typically achieved using two methods:
 There are two possible approaches to performing network forwarding for
 container-to-container traffic:
 
+Cluster Mesh
+============
+
+Cluster mesh extends the networking datapath across multiple clusters. It
```

Comment:
Nit: Is the official term `ClusterMesh` or `Cluster Mesh` or `cluster mesh`? It is inconsistent throughout the document - I think `ClusterMesh` is better because it makes it sound more cilium-specific ;)

Reply:
I've changed it to cluster mesh, it's what was concluded on the big servicemesh vs service mesh discussion.